department-of-veterans-affairs/caseflow
# Div/appeals 54281

Resolves [Navigation Updates: Existing Appeals](https://jira.devops.va.gov/browse/APPEALS-54281)

# Description
Implemented the ability to expand and collapse the existing appeals section,

## Acceptance Criteria

## Testing Plan


1. Login to caseflow with INBOUND_OPS_TEAM_ADMIN_USER
2. Click Your Queue
3. Select Correspondence Cases from switch view dropdown
4. Select any correspondence and redirect to correspondence details page.
5. Under **Correspondence and Appeals Tasks** tab Existing Appeals header should be collapsible with **+** and **-**  

[ ] For feature branches merging into master: Was this deployed to UAT?

# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1416" alt="Screenshot 2024-09-24 at 1 11 48 PM" src="https://github.com/user-attachments/assets/018ec93c-6409-4944-affe-cfe1f8f0ca6f">
<img width="1389" alt="Screenshot 2024-09-24 at 1 11 56 PM" src="https://github.com/user-attachments/assets/9abdb5f9-341e-400b-b23b-bca3dba1ccdc">


## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
